projects/1.20/assets/sons-of-sins/sons_of_sins/lang/zh_cn.json

```
@@ -0,0 +1,210 @@
+{
+  "advancements.a_heal_stick.descr": "使用装满骨灰的以太剑",
+  "advancements.a_heal_stick.title": "治愈法杖",
+  "advancements.ash_diamond.descr": "在血液中投掷16个灰烬",
+  "advancements.ash_diamond.title": "骨灰钻石",
+  "advancements.freddys_favorite_alloy.descr": "做出灵魂钢",
```

```suggestion
  "advancements.freddys_favorite_alloy.descr": "构建一块灵魂钢锭",
```
原文`Construct a Soul Steel Ingot`
但是我看前后文是没有锭的，具体是？